tests/cli/fixtures.go

```diff
@@ -7,33 +7,30 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/require"
-	tmtypes "github.com/tendermint/tendermint/types"
 
-	"github.com/cosmos/cosmos-sdk/std"
+	tmtypes "github.com/tendermint/tendermint/types"
 
 	"github.com/cosmos/cosmos-sdk/codec"
 	"github.com/cosmos/cosmos-sdk/server"
 	"github.com/cosmos/cosmos-sdk/simapp"
-)
-
-var (
-	cdc = std.MakeCodec(simapp.ModuleBasics)
+	"github.com/cosmos/cosmos-sdk/simapp/params"
 )
 
 // Fixtures is used to setup the testing environment
 type Fixtures struct {
-	BuildDir     string
-	RootDir      string
-	SimdBinary   string
-	SimcliBinary string
-	ChainID      string
-	RPCAddr      string
-	Port         string
-	SimdHome     string
-	SimcliHome   string
-	P2PAddr      string
-	Cdc          *codec.Codec
-	T            *testing.T
+	BuildDir       string
+	RootDir        string
+	SimdBinary     string
+	SimcliBinary   string
+	ChainID        string
+	RPCAddr        string
+	Port           string
+	SimdHome       string
+	SimcliHome     string
+	P2PAddr        string
+	Cdc            *codec.Codec
+	EncodingConfig params.EncodingConfig
```

Comment:
should we delete `Cdc` since it's included in the `EncodingConfig`?

Reply:
Since `Cdc` is used in other places in the tests, let's do that in a separate PR. The CLI tests will need to be upgrade to not use amino. Can you track that @anilCSE ?